docs/releases/2.23.0.md

```diff
@@ -0,0 +1,27 @@
+#### Release Links
+* lagoon [v2.23.0](https://github.com/uselagoon/lagoon/releases/tag/v2.23.0)
+* lagoon-ui [core-v2.23.0](https://github.com/uselagoon/lagoon-ui/releases/tag/core-v2.23.0)
+* lagoon-build-deploy [core-v2.23.0](https://github.com/uselagoon/build-deploy-tool/releases/tag/core-v2.23.0)
+* lagoon-core chart [1.50.0](https://github.com/uselagoon/lagoon-charts/releases/tag/lagoon-core-1.50.0)
+* lagoon-remote chart [0.96.0](https://github.com/uselagoon/lagoon-charts/releases/tag/lagoon-remote-0.96.0)
+
+## Release Notes
+
+This release is a standard Lagoon release. There are changes that teams managing their own Lagoon installations (core and remote) may need to be aware of.
+
+## Upgrades
+
+Self-managed Lagoon installs may need to be aware of these changes in lagoon-core.
+
+### Broker update to RabbitMQ 3.13
+This release of Lagoon includes an update to RabbitMQ 3.13 in https://github.com/uselagoon/lagoon/pull/3849 - which has necessitated a small modification to the feature flag enabler (to ignore experimental plugins like Khepri, that are known to have issues). As long as the upgrade runs in order successfully, the plugin should remain disabled.
+
+### KEYCLOAK_ADMIN_API_CLIENT_SECRET
+In https://github.com/uselagoon/lagoon/pull/3841 a new client was added to the Master realm in Keycloak to allow the API to configure Keycloak. This will remove the need for an admin user/password to be used, so MFA can be set on the admin account. The charts will autocreate and configure the secret if none is provided.
+
+## Deprecations
+
+Self-managed Lagoon installs may need to be aware of these changes in lagoon-remote.
+
+### Node sysctl daemonset replaces init containers for OpenSearch and Elasticsearch
+In https://github.com/uselagoon/build-deploy-tool/pull/395 a future release of the build deploy tool will remove the init containers from OpenSearch & Elasticsearch deployments that are used to set the necessary sysctl settings. This daemonset will run once on node startup to ensure the correct setting is in place for all future deployments. Ideally, the necessary configuration should be performed at the Infrastructure provisioning, in which case these daemonsets will no-op.
```

Comment:
This is disabled by default, it should call out that if they are leveraging opensearch or elasticsearch containers, that they will need to enable the feature in `lagoon-remote` values, see https://github.com/uselagoon/lagoon-charts/blob/main/charts/lagoon-remote/values.yaml#L459